packages/holocron/__tests__/holocronModule.spec.jsx

```diff
@@ -339,31 +338,28 @@ describe('holocronModule', () => {
     // Wait one cycle of the event loop since we can't retrieve a promise from initiateLoad
     await sleep(1);
 
-    return loadPromise
```

Comment:
Does this still need to be stored as a variable?

Reply:
I have opted to assert value of status, instead of using snapshot.